### PR TITLE
Add reserved missions to my profile list

### PR DESCRIPTION
**Pull request by Draxeytina**

### CODE REVIEW

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
## Hi @Draxeytina

## Highlights
App working perfectly ✔️

## REQUIRED CHANGES

##
Cheers and Happy coding!👏👏👏

File: src/components/MissionsProfile.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { useSelector } from 'react-redux';
+
+const MissionsList = () => {
+  const { missions } = useSelector((state) => state.missions);
+  const missionList = Object.keys(missions);
+  const reservedMissions = [];
+
+  if (missionList.length) {
+    missionList.forEach((item) => {
+      if (missions[item].reserved) {
```
Comment (NduatiKagiri) [inline src/components/MissionsProfile.js:11]: Please use the `filter` method as it is in the requirements to use it rather than `if`

Comment (Draxeytina) [reply src/components/MissionsProfile.js:11]: You are right, i had ignored that. I refactored the object in order to use the filter method.

Comment (Draxeytina) [reply src/components/MissionsProfile.js:11]: I think it's done now.

**Review by NduatiKagiri — APPROVED** [review]
##Hi @Draxeytina
## Highlights
App working perfectly ✔️

## Status: Approved ✔️
Your project is complete! There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉

##
Cheers and Happy coding!👏👏👏

**Pull request merged**